packages/epo-react-lib/src/atomic/Accordion/Accordion.stories.tsx

```diff
@@ -25,7 +25,7 @@ const meta: ComponentMeta<typeof Accordion> = {
 };
 export default meta;
 
-export const Default: ComponentStoryObj<typeof Accordion> = {
+export const Primary: ComponentStoryObj<typeof Accordion> = {
```

Comment:
renamed all "Default" to "Primary" to avoid confusion that the displayed component is always the default parameters of the component